jenkins-master/jobs/scripts/workspace/runtests.py

```diff
@@ -136,6 +136,10 @@ def query_args(self):
 
         return args
 
+class MediaRunner(BaseRunner):
+
+    def __init__(self, *args, **kwargs):
+        BaseRunner.__init__(self, *args, **kwargs)
```

Comment:
It's better to use explicit class names when calling methods from the base class. `super` should not really be used.